packages/datetime/src/index.ts

```diff
@@ -27,6 +27,7 @@ export { MonthAndYear } from "./common/monthAndYear";
 export type { DayPickerProps } from "./common/reactDayPickerProps";
 export type { TimePickerProps } from "./common/timePickerProps";
 export { TimePrecision } from "./common/timePrecision";
+export { TIMEZONE_ITEMS, MINIMAL_TIMEZONE_ITEMS } from "./common/timezoneItems";
```

Comment:
Do you think the names are crisp enough for an addition to the API? We could maybe consider condensing them to something like `TIMEZONES` and `MINIMAL_TIMEZONES`, maybe also TSDocs on the fields. What do you think?